packages/strapi-icons/package.json

```diff
@@ -25,7 +25,7 @@
     "react-dom": "^17.0.0 || ^18.0.0"
   },
   "scripts": {
-    "build": "yarn generate:icons && yarn build:prod && yarn generate:types",
```

Comment:
Can we add DTS=true to "build"/"generate:types" script to generate types directly from command?
e.g: "build": "yarn generate:icons && cross-env DTS=true yarn build:prod",

Also we are not using generate:types script anymore, should we remove/replace it?